nilearn/datasets/func.py

```diff
@@ -2126,6 +2126,75 @@ def fetch_development_fmri(n_subjects=None, reduce_confounds=True,
                  description=fdescr)
 
 
+def fetch_surf_tva_localizer(data_dir=None, verbose=1, resume=True):
+    """Download the data from one subject of the InterTVA dataset,
+        for the event-related voice localizer run.
+        The data provided here is a set of beta maps each estimated
+        for one of the 144 audio stimuli presented to the subject,
+        and projected onto the cortical surface in the fsaverage5
+        template space, for the left hemisphere.
+        It is used for the example of the cortex-based searchlight decoding.
+
+    Parameters
+    ----------
+    data_dir: str, optional
+        Path of the data directory. Used to force data storage in a specified
+        location. Default: None
+
+    verbose: int, optional (default 1)
+        Defines the level of verbosity of the output.
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        Dictionary-like object, the interest attributes are :
+         - 'func_left': Paths to Gifti files containing resting state
+                        time series left hemisphere
+         - 'phenotypic': array containing tuple with subject ID, age,
+                         dominant hand and sex for each subject.
```

Comment:
IIUC, it mostly (uniquerly ?) constains the labels information

Reply:
done! (sorry, there was some bad copy-paste on my part...)